types/tests.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from '@hapi/joi'
```

Comment:
This error happens when I opened this file on VSCode.

 Parsing error: `import' and 'export' may appear only with 'sourceType: module`

![image](https://user-images.githubusercontent.com/7704148/81637096-e661fb00-93eb-11ea-86bd-0207e2d61fe1.png)

Reply:
I don't think this error is related to `joi-extension-semver`.

Reply:
Ah, I misunderstood that this file is the `types.ts` checked into this module. Still, it seems to be an editor config issue of some kind.

Reply:
No problem. I also don't think so.

But when I saw it, I thought it could be fixed.